ionic2/app/components/map/map.component.ts

```diff
@@ -34,9 +35,14 @@ export class MapComponent {
     this.map.goTo(position);
   }
 
-  filter(filterOptions: FilterOptions) {
-    console.debug('map:filter', filterOptions);
-    this.map.filter(filterOptions);
+  filter(filter: Filter) {
+    let filterObj:FilterOptions = {
+      pokemonIds: filter.selectedPokemon,
+      sightingsSince: filter.sightingsRange,
+      predictionsUntil: filter.predictionsRange
+    };
+    console.debug('map:filter', filter);
```

Comment:
I will be clarifying things with PokeMap-1 later today. But this is something that doesn't need to be fixed in the pr. Since it wasn't marked with a todo before, I forgot to add it.